packages/types/src/base.ts

```diff
@@ -28,6 +28,17 @@ export enum KnownChainIds {
   ThorchainMainnet = 'cosmos:thorchain-mainnet-v1',
 }
 
+export type EvmChainId =
```

Comment:
consistency nitpick - declare utxo and cosmossdk chain id type here as well

Reply:
sound good, here ya go bd9b74e02d177062f32aec114e8d3f0dda5e90ff